packages/network-controller/src/NetworkController.ts

```diff
@@ -954,6 +976,20 @@ export class NetworkController extends BaseController<
       `${this.name}:getSelectedNetworkClient`,
       this.getSelectedNetworkClient.bind(this),
     );
+
+    this.messagingSystem.registerActionHandler(
```

Comment:
Could we add unit tests for these actions? Something that confirms that when the `addNetwork` messenger action is called, the add network behavior is triggered. We could either check that the `addNetwork` method is called or do something like copy a "basic" test such as this: https://github.com/MetaMask/core/blob/f47d5b47c5f33607fa4cb277987c2a2699d86d26/packages/network-controller/tests/NetworkController.test.ts#L3405. And similar for the `removeNetwork` messenger action.

Reply:
Yep sure, I can add some tests to validate the messaging system registrations.

Reply:
Tests have been added.